src/lightning_app/utilities/proxies.py

```diff
@@ -398,6 +399,9 @@ def run_once(self):
             )
             self.delta_queue.put(ComponentDelta(id=self.work_name, delta=Delta(DeepDiff(state, self.work.state))))
             self.work.on_exception(e)
+            print("########## CAPTURED EXCEPTION ###########")
```

Comment:
This is done to help to debug. Exception was not shown and users were confused as captured in the state.